docs/pages/web/utilities/defaultlabelprovider.tsx

```diff
@@ -275,7 +275,7 @@ function getLabelsTable(/* fallbackLabels: { [string]: { [string]: mixed } } */)
     {
       component: 'Toast',
       prop: 'accessibilityDismissButtonLabel',
-      label: 'Dismiss toast',
```

Comment:
let's make sure this gets updated in Pinboard!

Reply:
Doing it in https://github.com/pinternal/pinboard/pull/83102
Blocked by AI that insists I use Dismiss instead of Dismiss 🤖
I made a post on i18n to check about that.